api_server.py

```
@@ -80,32 +93,34 @@ def get_crew_list() -> List[Dict]:
-            display += end_time.strftime(" on %d/%m")
 
-        return {
-            "available": True,
-            "duration": format_hours(duration_min),
-            "end_time_display": display,
-        }
+def get_availability(entity_id: int, table: str, now: datetime) -> Dict:
+    col = "crew_id" if table == "crew_availability" else "appliance_id"
+    conn = get_db()
+    curr = conn.execute(
+        f"SELECT end_time FROM {table} WHERE {col} = ? AND start_time <= ? AND end_time > ? LIMIT 1",
```

Ruff (0.15.7)

[error] Possible SQL injection vector through string-based query construction

(S608)